tests/codegen/sanitizer/sanitizer-recover.rs

```diff
@@ -3,44 +3,44 @@
 //
 //@ needs-sanitizer-address
 //@ needs-sanitizer-memory
-//@ revisions:ASAN ASAN-RECOVER MSAN MSAN-RECOVER MSAN-RECOVER-LTO
+//@ revisions: asan asan-recover msan msan-recover msan-recover-lto
```

Comment:
Maybe this should be a separate PR?

Reply:
Good point; if this goes forward, I will split the upper->lowercase items out.